pkg/rtc/mediatracksubscriptions.go

```diff
@@ -304,7 +304,12 @@ func (t *MediaTrackSubscriptions) closeSubscribedTrack(subTrack types.Subscribed
 		return
 	}
 
-	dt.CloseWithFlush(!willBeResumed)
+	if willBeResumed {
+		dt.CloseWithFlush(!willBeResumed)
```

Comment:
nit: `CloseWithFlush(false)` requires slightly less mental gymnastics.

Reply:
yeah, sorry, the flipped sense is a bit of a mess. Will change to `false` and `true`.